server/game/cards/events/01/growingstrong.js

```diff
@@ -0,0 +1,41 @@
+const _ = require('underscore');
+
+const DrawCard = require('../../../drawcard.js');
+
+class GrowingStrong extends DrawCard {
+
+    canPlay() {
+        if(this.game.currentPhase !== 'challenge') {
+            return false;
+        }
+        return true;
+    }
+
+    play() {
+        this.game.promptForSelect(this.controller, {
+            numCards: 3,
+            activePromptTitle: 'Select Tyrell characters to buff',
```

Comment:
Odd, I thought i'd already commented on this.. oh well i'll do it again.  I'm not sure I like the word 'buff' as it's not a game term.  Can we perhaps just end this prompt at 'characters'?